index.html

```diff
@@ -23,21 +23,32 @@
                     <path fill-rule="evenodd"
                         d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z" />
                 </svg>
+                <div class="nav-list">
+                    <ul>
+                        <li><a href="#portfolio">Portfolio</a></li>
+                        <li><a href="#about">About me</a></li>
+                        <li><a href="#contact">Contact</a></li>
+                        <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-envelope-o"
+                                    aria-hidden="true"></i></a></li>
+                    </ul>
+                </div>
```

Comment:
Please kindly use the font properties (especially font-type) of the Figma design provided for you for this headline section 😄 

Figma design :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/1f35cdc9-cad8-4d39-b7b9-14672fcbd5c5)

Your design :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/c1bf414e-6553-48fa-8b82-3c6eb20e376b)